packages/postgres-database/src/simcore_postgres_database/migration/versions/feca36c8e18f_rename_tags_to_groups.py

```diff
@@ -0,0 +1,23 @@
+"""rename tags_to_groups
+
+Revision ID: feca36c8e18f
+Revises: e8057a4a7bb0
+Create Date: 2024-08-23 12:30:56.650085+00:00
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "feca36c8e18f"
+down_revision = "e8057a4a7bb0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.rename_table("tags_to_groups", "tags_access_rights")
```

Comment:
I actually prefer tags_access_rights than tags_to_groups. it is more straightforward. at least for me.

Reply:
@sanderegg but that is what I am doing here, right? :-) I am renaming from `tags_to_groups` TO `tags_access_rights` :-)

Reply:
yes I saw that and I like it.
:100: